frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -218,48 +223,70 @@ const TranslationPage = () => {
       </Box>
     ) : null;
   };
-
   return (
-    <Box margin="20px 20px 0px 20px">
-      <Text size="lg">Story Title Here</Text>
-      <Text as="ins">View story details</Text>
-      <FontSizeSlider setFontSize={handleFontSizeChange} />
-      <Flex>
-        <Flex direction="column" width="75vw">
-          <Flex direction="row">
-            <IconButton
-              variant="ghost"
-              aria-label="Undo Change"
-              onClick={undoChange}
-              icon={<Icon as={MdUndo} />}
-            />
-            <IconButton
-              variant="ghost"
-              aria-label="Redo Change"
-              onClick={redoChange}
-              icon={<Icon as={MdRedo} />}
-            />
+    <Box height="100vh" overflow="hidden">
+      <Header title={title} />
+      <Divider />
+      <Flex justify="space-between">
+        <Box minWidth="calc(100vw - 350px)" maxWidth="75vw">
+          <Flex justify="space-between" alignItems="center" margin="10px 30px">
+            <FontSizeSlider setFontSize={handleFontSizeChange} />
+            <Flex direction="row">
+              <IconButton
+                size="undoRedo"
+                variant="ghost"
+                aria-label="Undo Change"
+                onClick={undoChange}
+                icon={<UndoIcon />}
+              />
+              <IconButton
+                variant="ghost"
+                size="undoRedo"
+                aria-label="Redo Change"
+                onClick={redoChange}
+                icon={<RedoIcon />}
+              />
+            </Flex>
           </Flex>
-          {maxCharsExceededWarning()}
-          <TranslationTable
-            translatedStoryLines={translatedStoryLines}
-            onUserInput={onUserInput}
-            editable
-            fontSize={fontSize}
-          />
-        </Flex>
-        <Box margin="20px 30px 0 0">
-          <ProgressBar
-            percentageComplete={
-              (numTranslatedLines / translatedStoryLines.length) * 100
-            }
-          />
+          <Divider />
+          <Box margin="0 0 0 20px">
+            <Flex
+              direction="column"
+              overflowY="auto"
+              height="calc(100vh - 250px)"
+            >
+              {maxCharsExceededWarning()}
+              <TranslationTable
+                translatedStoryLines={translatedStoryLines}
+                onUserInput={onUserInput}
+                editable
+                fontSize={fontSize}
+                originalLanguage="English"
+                translatedLanguage={convertLanguageTitleCase(language)}
+              />
+            </Flex>
+            <Autosave
```

Comment:
nit: i kinda liked that AutoSave was at the bottom of the components since it's not a dom element in the usual sense, thoughts on moving it to the bottom again?